pkg/virt-controller/watch/clone/clone_test.go

```diff
@@ -686,6 +710,26 @@ var _ = Describe("Clone", func() {
 		})
 
 	})
+
+	Context("different sources", func() {
+		It("should support snapshot source", func() {
```

Comment:
should probably add another test of snapshot source in progress

Reply:
Done, you can find it under:
```go
Context("with source snapshot", func() {
  It("when snapshot is not ready yet - should not do anything", func() {
```